test/finnish-ssn_test.ts

```diff
@@ -65,7 +65,13 @@ describe('FinnishSSN', () => {
     })
 
     it('Should pass when given valid FinnishSSN from 20th century', () => {
-      expect(FinnishSSN.validate('010197-100P')).to.equal(true)
+      const hypotheticalIndividuals = [
+        '010101-0101',
```

Comment:
Added another one here just to be sure because most tests seemed to focus on SSNs around 1990-20xx